Watchman.Discord.UnitTests/Users/UsersControllerTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using AutoFixture.NUnit3;
+using Devscord.DiscordFramework.Commons;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Watchman.Discord.Areas.Users.BotCommands;
+using Watchman.Discord.UnitTests.TestObjectFactories;
+
+namespace Watchman.Discord.UnitTests.Users
+{
+    internal class UsersControllerTests
+    {
+        private readonly TestControllersFactory testControllersFactory = new();
+        private readonly TestContextsFactory testContextsFactory = new();
+
+        [Test, AutoData]
+        public async Task GetAvatar_ShouldDisplayUserAvatar(AvatarCommand command)
+        {
+            //Arrange
+            var userContext = testContextsFactory.CreateUserContext(1);
+            var contexts = testContextsFactory.CreateContexts(1, 1, 1);
+            var messagesServiceMock = new Mock<IMessagesService>();
+            messagesServiceMock.Setup(x => x.SendMessage(It.IsAny<string>(), It.IsAny<MessageType>()))
+               .Returns<string, MessageType>((a, b) => Task.FromResult(userContext.AvatarUrl));
+            var messagesServiceFactoryMock = new Mock<IMessagesServiceFactory>();
+            messagesServiceFactoryMock.Setup(x => x.Create(It.IsAny<Contexts>()))
+                .Returns(messagesServiceMock.Object);
+            var usersServiceMock = new Mock<IUsersService>();
+            usersServiceMock.Setup(x => x.GetUserByIdAsync(It.IsAny<DiscordServerContext>(), It.IsAny<ulong>()))
+                .Returns<DiscordServerContext, ulong>((a, b) => Task.FromResult(userContext));
```

Comment:
```suggestion
            usersServiceMock.Setup(x => x.GetUserByIdAsync(It.IsAny<DiscordServerContext>(), It.IsAny<ulong>()))
                .Returns(Task.FromResult(userContext));
```
Można to krócej zapisać. Zauważyłem, że taki "dłuższy zapis" występuje też w innych miejscach. Trzeba pamiętać o tym, że można to krócej zapisywać.

Reply:
Potwierdzam 
Ten dłuższy zapis ma sens, jak chcesz robić akcje zależne od danych wejściowych, jeśli chcesz na sztywno zwracać to samo niezależnie od danych na wejściu, samo Returns wystarczy 

zobacz jak to wygląda przykładowo tutaj 
https://github.com/Devscord-Team/Watchman/blob/master/Watchman.Discord.UnitTests/Messaging/SendControllerTests.cs

Reply:
Ta kwestia nadal jest nie rozwiązana!